app/javascript/helpers/Axios.jsx

```diff
@@ -11,6 +11,7 @@ export const ENDPOINTS = {
   changePassword: '/users/change_password.json',
   admin: {
     getRoles: 'admin/roles.json',
+    createRole: 'admin/roles.json',
```

Comment:
Couldn't comes up with a good naming for the `admin/roles.json` endpoint.
Do you like `roles` and differentiate only through the request method `POST|GET|...`?